slides/syntroduction/slides.adoc

```diff
@@ -229,16 +226,21 @@ Important tools - integral part of Project Syn:
 
 Let's have a look at some details.
 
-== Deep Dive: Commodore Hierarchy (Reclass)
+== Deep Dive: Commodore Hierarchy
 
-* Component defaults
-* Global generic
-* Kubernetes distribution
-* Cloud provider
-* Cloud provider region
-* Customer- and cluster-specific configuration
+. Component defaults
+. Global generic
+. Kubernetes distribution
+. Cloud provider
+. Cloud provider region
+. Customer- and cluster-specific configuration
 
-(currently hard-coded)
+This is based on https://github.com/kapicorp/reclass[reclass].
+
+[.notes]
+--
+* This structure is currently hard-coded
```

Comment:
This is currently being changed. It might make sense to already reflect that change here.